core/processor/src/main/java/io/quarkus/annotation/processor/ExtensionAnnotationProcessor.java

```diff
@@ -124,7 +124,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
     @Override
     public Iterable<? extends Completion> getCompletions(Element element, AnnotationMirror annotation, ExecutableElement member,
-            String userText) {
+                                                         String userText) {
```

Comment:
This does not look like the formatter plugin was used, or?

Reply:
yes i think i make a mistake using `CTRL + ALT + L` on Intelij.
i need revert this?